sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -75,6 +75,10 @@
     "region = session.boto_region_name\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    container = retrieve("xgboost", region, version="1.2-1")
</pre><p>current recommendation is to use 1.2-2 or later... is that going to be a problem here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I don't think is a problem, thanks for noticing. I think we can upgrade in a different PR.</p>